spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -102,11 +102,11 @@ Note that the sum of `ft.<name>.timeout.callsTimedOut.total` and `ft.<name>.time
 
 |`ft.<name>.circuitbreaker.callsSucceeded.total`
 | Counter | None
-| Number of calls allowed to run by the circuit breaker that returned successfully
+| Number of calls allowed to run by the circuit breaker that were considered a success
```

Comment:
I was thinking if this should read "that were considered a success _by the circuit breaker_" (dtto for the failure case below), but I thought that's perhaps too repetitive. WDYT?

Reply:
Hmm, I see what you mean but it is a bit repetitive. How about:
* Number of calls which were allowed to run and were considered a success by the circuit breaker
* Number of calls which were allowed to run and were considered a failure by the circuit breaker
* Number of calls which were prevented from running by the circuit breaker _(slight change to the current wording)_

Reply:
That's nice, thanks!